scripts/gceworker.sh

```diff
@@ -73,16 +89,14 @@ case "${cmd}" in
     ;;
     start)
     start_and_wait "${NAME}"
-    if ! gcloud compute config-ssh > /dev/null; then
-	echo "WARNING: Unable to invoke config-ssh, you may not be able to 'ssh ${FQNAME}'"
-    fi
+    refresh_ssh_config
 
     # SSH into the node, since that's probably why we started it.
     echo "****************************************"
     echo "Hint: you should also be able to directly invoke:"
-    echo "ssh ${FQNAME}"
+    echo "ssh <USERNAME_DOMAIN_SUFFIX>@${FQNAME}"
```

Comment:
Could you change this to something that people can copy paste? I have no idea what I would need to replace `<USERNAME_DOMAIN_SUFFIX>` with.

On this PR, I'm able to "just run" `(mosh|ssh) gceworker-tbg.us-east1-b.cockroach-workers` and it works. So maybe that is what this (and below) can be changed to?

Reply:
Did you try to recreate your gceworker using this script? This only affects newly created gceworkers (ie you have to run gceworker.sh create). I checked your VM and it looks like you were able to connect using ssh gceworker-tbg.us-east1-b.cockroach-workers because that VM still uses the project metadata ssh public key. Once this PR is merged and we convert any running GCEworkers to use OS login you won't be able to just run `(mosh|ssh) gceworker-tbg.us-east1-b.cockroach-workers` . You will have to run something like `ssh tobias_cockroachlabs_com@gceworker-tbg.us-east1-b.cockroach-workers` . I will include more documentation and example on what exactly USERNAME_DOMAIN_SUFFIX is.